pages/techgenius/tg-hardware-setup.md

```diff
@@ -29,9 +27,8 @@ Note: We recommend that talk to OLE technical staff before purchasing a router,
 
 Connect these two devices (router and RPi) via two cables - a micro-USB to USB (on the left) and an Ethernet cable (RJ45) (on the right).
 
-![RJ45 Cable](images/tg-quickstart-cable-rj45.png)
+![USB A > C Cable](images/tg-quickstart-cable-usb.png) ![RJ45 Cable](images/tg-quickstart-cable-rj45.png)
```

Comment:
By the way, could we update `USB A > C Cable` to `USB A to Micro-B Cable` here?

Reply:
Noted